client/src/components/TestSuite/TestSuiteDetails/TestGroupCard.tsx

```diff
@@ -59,7 +60,9 @@ const TestGroupCard: FC<TestGroupCardProps> = ({
       </div>
       {view === 'run' && description && (
         <>
-          <Box margin="20px">{description}</Box>
+          <Box margin="20px">
+            {runnable.description && runnable.description.length > 0 && description}
+          </Box>
           <Divider />
         </>
       )}
```

Comment:
I think this needs to be refactored to move the conditional up -- box and divider should not render if there is no description